Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceUpdateExecutorInterface.h

```diff
@@ -22,7 +22,7 @@ namespace AzToolsFramework
             AZ_RTTI(InstanceUpdateExecutorInterface, "{26BB4ACF-6EA5-4668-84E7-AB75B90BA449}");
             virtual ~InstanceUpdateExecutorInterface() = default;
 
-            // Add all Instances of Template with given Id into a queue for updating them later.
+            // Add all Instances of Template with given Id into a queue so it's updated at the end of the frame.
```

Comment:
nit: since this can be spread across multiple frames in the future, I suggest reverting these newly added comments.